docs/architecture/decisions/0012-non-govuk-domain-policy.md

```diff
@@ -0,0 +1,38 @@
+# 12. Non-GOV.UK domain policy
+
+Date: 2025-05-06
```

Comment:
Shouldn't this be the original date of 2022-10-10 ?

Reply:
I changed it as an opportunity to re-review the doc with today's date. Thought it might throw things off a bit if these ADRs aren't in date order. I'm happy leaving it as today's date if you are 👍